### PR TITLE
Audit - Game Projects Weeks 4 & 5

### PR DESCRIPTION
**Pull request by jsnorek**
This is a PR for issue #2114 : Auditing the projects in Week 4 & 5 and implementing minor changes to the projects instructions for understanding and clarity. 

In Week 4, I added text to clarify the Github deployment extension item as it read a bit clunky. I also changed the header to "Optional Extension" to stay consistent.

Screenshot 2024-12-12 at 9 03 55 AM

In Week 5, I added some context to the linked example problem as they are from an outdated version of React. I noted this and also added a link for the new React documentation as well as a Tic-Tac-Toe tutorial from React.

Screenshot 2024-12-12 at 9 04 26 AM

The rest of the content in both weeks feels clear, is readable, and leaves a lot of room for creativity when choosing a game to build.

### CODE REVIEW

**Review by daaimah123 — CHANGES_REQUESTED** [review]
Great start, I especially like your note about the outdated content with reference to up to date options! 

I see you've attached this PR to the proper project and updated its status! 👏🏾 

You are some requirements from [your PR requirement](https://docs.google.com/document/d/1Yx2S5xgnpJZmZRu5MCkDtbIcLnmOg84Rr5PpfQCqlRo/edit?tab=t.b57zbfc3ko9k#heading=h.tboe7r2qx4wg):
- Reference the issue in the “Development” section on the right hand pane of your PR (this will enable both the issue and your PR to be connected in your Kanban board and automatically trigger processes such as closing the issue at the same time your PR is merged in)
- You have your objective clearly stated, but a **_description_** of your issue is missing:
  - without me physically clicking into your issue, I do not know what problem you are solving and why, this can delay your feedback iteration time if a developer has to take the extra time to find your issue

Best Practice: 
- Give a slack nudge to your reviewer to let them know that you have request them as a reviewer on your PR

**Comment by jsnorek** [comment]
@daaimah123 I have referenced the issue in "development" and added a description to this PR :)

File: projects/js-html-games.md
Comment (ChasVanDav) [inline projects/js-html-games.md:1]: This was a great clarification to the instructions.

**Review by ChasVanDav — COMMENTED** [review]
@jsnorek Great job on updating the instructions to be more clear and up-to-date!

**Review by daaimah123 — APPROVED** [review]

**Comment by daaimah123** [comment]
@jsnorek I have approved your changes. Your audit looks good! Please let me know if you have any additional incoming changes. If not, go ahead and "Merge pull request" and leave a respective comment on your GH issue closing it.

**Comment by jsnorek** [comment]
Thank you @ChasVanDav @daaimah123 
I'll merge now :)

**Pull request merged**